src/fr-device/sample-profile.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Sample, generated profile from Edge on Mac. This will be removed once feature is finalized.